core/Decode.hpp

```diff
@@ -120,6 +123,15 @@ namespace olympia
             //! \brief ...
             PARAMETER(FileNameListType, fusion_group_definitions, {},
                       "Lists of fusion group UID json files")
+
+            //! LMUL
```

Comment:
Since we allow vset instructions to change these values, can we rename these params to something like `init_lmul`, `init_sew` and `init_vl`?

Reply:
Renamed in [73ca9f4](https://github.com/riscv-software-src/riscv-perf-model/pull/167/commits/73ca9f4adb0bce0b8756a9b1ab3d88bf96c589bd)